Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM fedora:28
+LABEL \
+    name="ResultsDB-Updater application" \
+    vendor="ResultsDB-Updater developers" \
+    license="LGPL-2.1" \
+    description="ResultsDB-Updater is a micro-service that listens for test \
+results on the CI message bus and updates ResultsDB in a standard format." \
+    usage="https://github.com/release-engineering/resultsdb-updater" \
+    build-date=""
+
+RUN dnf install -y \
+        fedmsg \
+        python-moksha-hub \
+        python-fedmsg-commands \
+        python-fedmsg-consumers \
+        fedmsg-hub \
```

Comment:
I don't think this dependency is required if you install `fedmsg` as you do above.